src/rgbdscheck.asm

```diff
@@ -0,0 +1,7 @@
+; Abort if RGBDS version is too low
+IF(!DEF(__RGBDS_MAJOR__) || !DEF(__RGBDS_MINOR__) || !DEF(__RGBDS_PATCH__))
+FAIL "Requires RGBDS version >= 0.5.1"
+ENDC
+IF((__RGBDS_MAJOR__ == 0 && __RGBDS_MINOR__ < 5) || (__RGBDS_MAJOR__ == 0 && __RGBDS_MINOR__ == 5 && __RGBDS_PATCH__ < 1))
```

Comment:
I know this is a long line, but old versions of RGBDS didn't support `\` line continuations.